playbooks/cloud-provider-openstack-acceptance-test-keystone-authentication-authorization/run.yaml

```diff
@@ -15,8 +15,10 @@
           set -e
           set -o pipefail
 
+          export ARCH=${ARCH:-amd64}
+          export BUILD_CMDS='k8s-keystone-auth openstack-cloud-controller-manager'
```

Comment:
No need to build openstack-cloud-controller-manager

Reply:
We expose openstack-cloud-controller-manager binary with
`export EXTERNAL_CLOUD_PROVIDER_BINARY="$PWD/openstack-cloud-controller-manager"` at line 77, we can remove it here if EXTERNAL_CLOUD_PROVIDER_BINARY is not required

Reply:
ok, let's keep it, removing EXTERNAL_CLOUD_PROVIDER_BINARY in not in scope of this PR.